.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: AI Code Reviewer
+
+on: [pull_request]
+
+    # types:
+    #   - opened
+    #   - synchronize
+# permissions: write-all
```

Comment:
The `permissions` field is commented out. If you want to set the permissions for the workflow, you should uncomment this line.